packages/PHPStanStaticTypeMapper/TypeMapper/NeverTypeMapper.php

```diff
@@ -26,7 +26,7 @@ public function getNodeClass(): string
      */
     public function mapToPHPStanPhpDocTypeNode(Type $type): TypeNode
     {
-        return new IdentifierTypeNode('mixed');
+        return new IdentifierTypeNode('never');
```

Comment:
this change was necessary, because otherwise the call to 

```
        $phpDocInfo = $this->phpDocInfoFactory->createFromNodeOrEmpty($node);
        $this->phpDocTypeChanger->changeReturnType($phpDocInfo, new NeverType());
```

inserted a `mixed` type, instead of the expected `never` type.

Reply:
hmm turns out this change makes the test-suite fail, because it expects the `mixed` type.

is there any other way I could put the phpdoc of "@return never" into the AST?

Reply:
with https://github.com/rectorphp/rector-src/pull/296/commits/3eba837356c6d233c95ef07d558c3e8e6b3fcd1d I tried a different approach. it seems to work and does not make already existing tests fail.

its not nice, but work.. lets see what you guys think